packages/app/src/components/catalog/Component.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import {
+  GitHubActionsPlugin,
+  GITHUB_ACTIONS_ANNOTATION,
+} from '@backstage/plugin-github-actions';
+import {
+  useEntity,
+  EntityPageTabs as Tabs,
+  EntityMetadataCard,
+} from '@backstage/plugin-catalog';
+import { Grid } from '@material-ui/core';
+
+const OverviewPage = () => {
+  const entity = useEntity();
+  return (
+    <Grid item sm={4}>
+      <EntityMetadataCard entity={entity} />
+    </Grid>
+  );
+};
+
+// Just for illustration purposes, gonna live in plugin-circle-ci
+const CIRCLE_CI_ANNOTATION = 'circle-ci-slug-dummy';
+
+const DefaultEntity = () => {
+  const entity = useEntity();
+
+  const isCIAvailable = [
+    entity!.metadata!.annotations?.[GITHUB_ACTIONS_ANNOTATION],
+    entity!.metadata!.annotations?.[CIRCLE_CI_ANNOTATION],
+  ].some(Boolean);
+
+  return (
+    <Tabs>
+      <Tabs.Tab title="Overview" path="/" exact>
```

Comment:
the exact thing can be deprecated under react-router 6 right?

Reply:
This is not the react-router `exact` though, it's part of the Tabs implementation

Reply:
Sure but maybe we want to change the tab implementation.